src/Trakx.Circle.ApiClient.Tests/Integration/BankAccountClientTest.cs

```diff
@@ -0,0 +1,104 @@
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+
+namespace Trakx.Circle.ApiClient.Tests.Integration;
+
+/// <summary>
+/// Bank account client test
+/// <see cref="BankAccountClientTest"/>
+/// </summary>
+public class BankAccountClientTest : CircleClientTestsBase
+{
+    private readonly IBankAccountsClient _bankAccountsClient;
+    private readonly IPaymentsClient _paymentsClient;
+    private readonly MockCreators _mockCreator;
+    /// <summary>
+    /// constructor for <see cref="BankAccountClientTest"/>
+    /// </summary>
+    /// <param name="apiFixture"></param>
+    /// <param name="output"></param>
+    public BankAccountClientTest(CircleApiFixture apiFixture, ITestOutputHelper output) : base(apiFixture, output)
+    {
+        _bankAccountsClient = ServiceProvider.GetRequiredService<IBankAccountsClient>();
+        _paymentsClient = ServiceProvider.GetRequiredService<IPaymentsClient>();
+        _mockCreator = new MockCreators(output);
+        Logger.Information($"Instantiating  {nameof(BankAccountClientTest)}");
+    }
+
+    /// <summary>
+    /// creating US bank account with valid payload should be successful
+    /// <remarks>
+    /// The documentation state that the response code should be 201, but the sandbox response returned 200
+    /// either 200 or 201 should be valid
+    /// </remarks>
+    /// </summary>
+    [Fact]
+    public async Task Create_Us_Bank_Account_Should_be_Successful()
+    {
+        Logger.Information("Running test for creating new Us Bank");
+        var bankRequest = _mockCreator.WireCreationRequestUs();
+        var bankCreated =  await _bankAccountsClient.CreateWireBankAccountAsync(bankRequest);
+
+        bankCreated.Should().NotBeNull();
+        bankCreated?.Result.Data.Id.Should().NotBeEmpty();
+        int[] expectedCode = { StatusCodes.Status200OK, StatusCodes.Status201Created };
+        Logger.Information("Us bank was created successfully with {DataId}", bankCreated?.Result.Data.Id);
+        Assert.Single(expectedCode, bankCreated?.StatusCode);
```

Comment:
Please use FluentAssertions, that might be my fault, but I don't even know what that `.Single` method does, and would have to look in the code to find out.